models/fivetran_log__usage_mar_destination_history.sql

```diff
@@ -47,13 +49,15 @@ join_usage_mar as (
         destination_mar.destination_name,
         usage.credits_spent,
         usage.dollars_spent,
-        destination_mar.monthly_active_rows,
+        destination_mar.free_monthly_active_rows,
+        destination_mar.paid_monthly_active_rows,
+        destination_mar.total_monthly_active_rows,
 
         -- credit and usage mar calculations
-        round( cast(nullif(usage.credits_spent,0) * 1000000.0 as {{ dbt.type_numeric() }}) / cast(nullif(destination_mar.monthly_active_rows,0) as {{ dbt.type_numeric() }}), 2) as credits_spent_per_million_mar,
-        round( cast(nullif(destination_mar.monthly_active_rows,0) * 1.0 as {{ dbt.type_numeric() }}) / cast(nullif(usage.credits_spent,0) as {{ dbt.type_numeric() }}), 0) as mar_per_credit_spent,
-        round( cast(nullif(usage.dollars_spent,0) * 1000000.0 as {{ dbt.type_numeric() }}) / cast(nullif(destination_mar.monthly_active_rows,0) as {{ dbt.type_numeric() }}), 2) as amount_spent_per_million_mar,
-        round( cast(nullif(destination_mar.monthly_active_rows,0) * 1.0 as {{ dbt.type_numeric() }}) / cast(nullif(usage.dollars_spent,0) as {{ dbt.type_numeric() }}), 0) as mar_per_amount_spent
+        round( cast(nullif(usage.credits_spent,0) * 1000000.0 as {{ dbt.type_numeric() }}) / cast(nullif(destination_mar.total_monthly_active_rows,0) as {{ dbt.type_numeric() }}), 2) as credits_spent_per_million_mar,
+        round( cast(nullif(destination_mar.total_monthly_active_rows,0) * 1.0 as {{ dbt.type_numeric() }}) / cast(nullif(usage.credits_spent,0) as {{ dbt.type_numeric() }}), 0) as mar_per_credit_spent,
+        round( cast(nullif(usage.dollars_spent,0) * 1000000.0 as {{ dbt.type_numeric() }}) / cast(nullif(destination_mar.total_monthly_active_rows,0) as {{ dbt.type_numeric() }}), 2) as amount_spent_per_million_mar,
+        round( cast(nullif(destination_mar.total_monthly_active_rows,0) * 1.0 as {{ dbt.type_numeric() }}) / cast(nullif(usage.dollars_spent,0) as {{ dbt.type_numeric() }}), 0) as mar_per_amount_spent
```

Comment:
I'm curious on if you think we can add additional value here by including metrics such as how much **paid** MAR are you getting on the dollar (ex: Total Paid MAR / Total Spent). Previously we did not have access to report on this metric since all MAR was grouped together, but now that we can report on these metrics it may be worth doing.

Reply:
Hey @elanfivetran would be interested to chat more about this.